styles/rwgps/roads.mss

```diff
@@ -40,7 +40,7 @@ as well. */
     line-color: @trunk_fill; }
   [type='primary'] { line-color: @primary_fill; }
   [type='secondary'] { line-color: @secondary_fill; }
-  [type='tertiary'] { line-color: @tertiary_fill; }
+  //[type='tertiary'] { line-color: @tertiary_fill; }
```

Comment:
👍  nice catch, just caught the same thing. seriously trimming down road rendering/querying at low zoom levels will mirror google and gain big performance broners.